scripts/mas_sso.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+docker run -d -p 8180:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin \
```

Comment:
The user is now `admin` and the password is  `admin`?

Reply:
Just for testing, we can even skip setting up the username/password.